src/main/resources/CodyBundle.properties

```diff
@@ -175,6 +175,8 @@ notification.general.cody-not-running.title=Cody is starting...
 notification.general.cody-not-running.detail=Please wait a few seconds. If Cody still won't start, check the IDE logs for errors.
 notifications.edits.editing-not-available.title=Cody can't edit the file
 notifications.edits.editing-not-available.detail=This could be because the file is not writable or Cody has trouble communicating with the editor.
+notifications.cody-connection-timeout.title=Cody's connection timeout
+notifications.cody-connection-timeout.detail=Failed to start Cody in timely manner, please run any Cody action or restart Cody to retry.
```

Comment:
"Cody took longer than expected to start. Please run any Cody action or restart Cody to retry"

Reply:
Everything else is good!